lib/util/support.js

```diff
@@ -16,6 +17,11 @@ const SUPPORTED_ANDROID_BROWSERS = [
 const SUPPORTED_IOS_BROWSERS = [
   'safari'
 ];
+const SUPPORTED_IOS_THIRD_PARTY_BROWSERS = [
```

Comment:
This variable name could be better... "safari" isn't a third-party iOS browser 🙃

Reply:
Theoretically all browsers should be supported now. Since they are all Webkit based. I wonder if we go and download 10 third party browsers and try them would they all work? If yes, maybe in the PR we just allow all.

Reply:
But as I think more. They are less likely to take a PR that blindly adds all. So nvm.